cargo-pgrx/src/command/install.rs

```diff
@@ -364,21 +364,21 @@ fn copy_sql_files(
         crate::command::schema::generate_schema(
             user_manifest_path,
             user_package,
-            &package_manifest_path,
+            package_manifest_path.as_ref(),
             profile,
             is_test,
             features,
             target,
             Some(&dest),
-            Option::<String>::None,
+            None,
```

Comment:
This is a perfect example of what I mean by "using `impl AsRef` everywhere weakens type inference". It can't even infer the type of an Option, here!